build/tc479297/test-template-dockerbuild.json

```diff
@@ -41,7 +41,7 @@
           "dockerStrategy": {
             "from": {
               "kind": "DockerImage",
-              "name": "registry.access.redhat.com/rhscl/ruby-22-rhel7:latest"
+              "name": "registry.access.redhat.com/rhscl/ruby-25-rhel7:latest"
```

Comment:
Please change to use openshift imagestream like above, if you use this image directly, the case will be failed in disconnected cluster; but if use openshift imagestream, we can mirror/import ruby image.